### ISSUE TITLE
-Wmissing-braces in drivers/tee/optee/device.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#370]
```
drivers/tee/optee/device.c:39:31: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        struct tee_param param[4] = {0};
                                     ^
                                     {}
drivers/tee/optee/device.c:92:48: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        struct tee_ioctl_open_session_arg sess_arg = {0};
                                                      ^
                                                      {}
2 warnings generated.
```

Patch sent: https://lore.kernel.org/lkml/20190219091520.28990-1-natechancellor@gmail.com/

**Comment by nickdesaulniers** [comment]
accepted: https://git.kernel.org/pub/scm/linux/kernel/git/arm/arm-soc.git/commit/?h=arm/drivers&id=50ceca6894adb834126ae644920902473bdc2d33

**Comment by nathanchance** [comment]
Fixed in `next-20190221`.
